src/status_im2/subs/wallet/wallet_test.cljs

```diff
@@ -84,66 +77,63 @@
           :operable                 "fully"
           :mixedcase-address        "0x7bcDfc75c431"
           :public-key               "0x04371e2d9d66b82f056bc128064"
-          :removed                  false}})
+          :removed                  false
+          :tokens                   tokens-0x2}})
 
 (h/deftest-sub :wallet/balances
   [sub-name]
-  (testing "returns seq of maps containing :address and :balance"
-    (swap! rf-db/app-db #(-> %
-                             (assoc-in [:wallet :accounts] accounts)
-                             (assoc :wallet/tokens tokens)))
-    (is (= `({:address "0x1"
-              :balance 3250}
-             {:address "0x2"
-              :balance 2100})
+  (testing "a map: address->balance"
+    (swap! rf-db/app-db #(assoc-in % [:wallet :accounts] accounts))
+
+    (is (= {"0x1" 3250 "0x2" 2100}
            (rf/sub [sub-name])))))
 
 (h/deftest-sub :wallet/accounts
   [sub-name]
   (testing "returns all accounts without balance"
-    (swap! rf-db/app-db
-      #(-> %
-           (assoc-in [:wallet :accounts] accounts)
-           (assoc :wallet/tokens tokens)))
+    (swap! rf-db/app-db #(assoc-in % [:wallet :accounts] accounts))
+
     (is
-     (= `({:path                     "m/44'/60'/0'/0/0"
```

Comment:
We were using ``` ` ``` instead of `'` to quote lists. It's unnecessary, actually, if we try to add a var inside, (like I did for the `:tokens`  key), we will need to also evaluate inside (using `~`). So changed the backticks to just `'`